.changeset/sharp-starfishes-compete.md

```diff
@@ -0,0 +1,20 @@
+---
+'astro': major
+---
+
+In the Integration API, the `app.render()` method of the `App` class has been simplified.
+
```

Comment:
I would rephrase this or add a note that this change is really only relevant for adapter/integration maintainers, not a something that users of official Astro adapters need to update.

Reply:
Updated in the commit titled "[clarify changeset](https://github.com/withastro/astro/pull/9199/commits/07ab063e657c64931961fa7a30ea60231ba5f23a)".